README.md

```diff
@@ -42,13 +44,14 @@ services:
 ```
 
 2) Convert OpenCompose file to Kubernetes objects
+
 ```sh
 opencompose convert -f hello-nginx.yaml
 ```
-This will create two files in current directory - *deployment-helloworld.yaml* and *service-helloworld.yaml*.
-First file is a Kubernetes Deployment object and second is a Kubernetes Service.
 
-To deploy your application to Kubernetes cluster just run `kubectl create -f service-helloworld.yaml -f deployment-helloworld.yaml`
+This will create two Kubernetes files in current directory - `deployment-helloworld.yaml` and `service-helloworld.yaml`.
+
+To deploy your application to Kubernetes run - `kubectl create -f service-helloworld.yaml -f deployment-helloworld.yaml`
```

Comment:
Ok, let's leave it as it was. I have overlooked and thought there this line was changed and that there two `kubectl create` calls before - hence my comment about more simplification.

What I would like here is for it to change from inline to:
```bash
kubectl create -f service-helloworld.yaml -f deployment-helloworld.yaml
```

It makes the line easier to copy-paste and IMO looks nicer